src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java

```diff
@@ -264,17 +257,13 @@ public static boolean isRemoveAirThatCanNotLand(final GameData data) {
   /**
    * Resets unit state, such as movement, submerged, transport unload/load, airborne, etc. Normally does not occur.
    */
-  public static boolean isResetUnitStateAtStart(final GameData data) {
+  static boolean isResetUnitStateAtStart(final GameData data) {
     final boolean isReset;
     data.acquireReadLock();
     try {
       final String prop =
-          data.getSequence().getStep().getProperties().getProperty(GameStep.PROPERTY_resetUnitStateAtStart);
-      if (prop != null) {
-        isReset = Boolean.parseBoolean(prop);
-      } else {
-        isReset = false;
-      }
+        data.getSequence().getStep().getProperties().getProperty(GameStep.PROPERTY_resetUnitStateAtStart);
+      isReset = prop != null && Boolean.parseBoolean(prop);
```

Comment:
This is kind of funky to read. Maybe add parentheses:
'isReset = (prop != null) && Boolean.parseBoolean(prop);'